.github/workflows/cd.yml

```diff
@@ -0,0 +1,38 @@
+name: NAME-EXAMINATION UI CD
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - "web/**"
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: "Environment"
+        required: true
+        type: choice
+        options:
+        - dev
+        - test
+        - sandbox
+        - prod
+      tagname:
+        description: "Specify a previous version (git tag) to deploy"
+        required: false
+        default: ""
+
+jobs:
+  analytics-ui-cd:
+    permissions:
+      contents: 'write'
+      id-token: 'write'  
+    uses: bcgov/name-examination/.github/workflows/cd.yaml@main
```

Comment:
should be bcgov/bcregistry-sre/.github/workflows/ui-cd.yaml@main

Reply:
Hmm, since this workflow is for name-examination, shouldn't it be the address currently written?